pygeo/parameterization/DVGeo.py

```diff
@@ -624,6 +660,68 @@ def addPointSet(self, points, ptName, origConfig=True, **kwargs):
             undeformed or deformed configuration. This should almost
             always be True except in circumstances when the user knows
             exactly what they are doing.
+        coord_xfer : function
+            A callback function that performs a coordinate transformation
+            between the DVGeo reference frame and any other reference
+            frame. The DVGeo object uses this routine to apply the coordinate
+            transformation in "forward" and "reverse" directions to go between
+            the two reference frames. Derivatives are also adjusted since they
+            are vectors coming into DVGeo (in the reverse AD mode)
+            and need to be rotated. We have a callback function here that lets
+            the user to do whatever they want with the coordinate transformation.
+            The function must have the first positional argument as the array that is
+            (npt, 3) and the two optional arguments that must be available are "mode"
+            ("fwd" or "bwd") and "apply_displacement" (True or False). This function
+            can then be passed to DVGeo through something like ADflow, where the
+            set DVGeo call can be modified as:
+            CFDSolver.setDVGeo(DVGeo, pointSetKwargs={"coord_xfer": coord_xfer})
+
+            An example function is as follows:
+
+            .. highlight:: python
+            .. code-block:: python
```

Comment:
Are both of these needed?

Reply:
removed one. lets see. I think it's a copy paste from somewhere else like most of my code